numpy/lib/nanfunctions.py

```diff
@@ -548,7 +551,7 @@ def nanprod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     Parameters
     ----------
     a : array_like
-        Array containing numbers whose sum is desired. If `a` is not an
+        Array containing numbers whose product is desired. If `a` is not an
         array, a conversion is attempted.
```

Comment:
Thanks @madphysicist, change made by `8fd1ee8` fixes the typo.  However, for your second comment-- I am not able to get the behavior you suggest for `np.prod` with `axis`, e.g.,

```
In [1]: import numpy as np

In [2]: a = np.random.randn(10,10)

In [3]: np.prod(a,axis=(1,2))
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-56314d82b79d> in <module>()
----> 1 np.prod(a,axis=(1,2))

/Users/pwolfram/anaconda/lib/python2.7/site-packages/numpy/core/fromnumeric.pyc in prod(a, axis, dtype, out, keepdims)
   2544 
   2545     return _methods._prod(a, axis=axis, dtype=dtype,
-> 2546                           out=out, **kwargs)
   2547 
   2548 

/Users/pwolfram/anaconda/lib/python2.7/site-packages/numpy/core/_methods.pyc in _prod(a, axis, dtype, out, keepdims)
     33 
     34 def _prod(a, axis=None, dtype=None, out=None, keepdims=False):
---> 35     return umr_prod(a, axis, dtype, out, keepdims)
     36 
     37 def _any(a, axis=None, dtype=None, out=None, keepdims=False):

ValueError: 'axis' entry is out of bounds
```

Am I missing something?